synedrion/src/uint/signed.rs

```diff
@@ -492,17 +462,6 @@ where
     }
 }
 
-impl<T> Add<&Signed<T>> for Signed<T>
```

Comment:
Was this no longer needed?

Reply:
I added the arithmetic ops (and `PartialOrd` and a few others) this summer; I used them in tests I think and also sort of thought it should be a "complete" number type, but I'm fine just implementing what we need too.

On a tangential note: do you think we will be able to get rid of our `Signed` impl at some point, given that `crypto-bigint` now supports it?